javalin/src/main/java/io/javalin/plugin/json/JavalinJackson.kt

```diff
@@ -52,8 +52,11 @@ class JavalinJackson(private var objectMapper: ObjectMapper? = null) : JsonMappe
 
     companion object {
         fun defaultMapper(): ObjectMapper = ObjectMapper()
+                // critical dependencies
                 .registerOptionalModule(OptionalDependency.JACKSON_KT.testClass)
                 .registerOptionalModule(OptionalDependency.JACKSON_JSR_310.testClass)
+                // extra dependencies
+            .registerOptionalModule(OptionalDependency.JACKSON_KTORM.testClass)
```

Comment:
Indentation should be the same here as above

Reply:
fixed!